axelrod/strategies/axelrod_second.py

```diff
@@ -199,3 +202,214 @@ def strategy(self, opponent: Player) -> Action:
         else:
             # Play TFT
             return opponent.history[-1]
+
+class Tranquilizer(Player):
+   
+    """
+    Submitted to Axelrod's second tournament by Craig Feathers
+
```

Comment:
Let's start with this:

```
Description given in Axelrod's "More Effective Choice in the 
Prisoner's Dilemma" paper: The rule normally cooperates but 
is ready to defect if the other player defects too often. 
Thus the rule tends to cooperate for the first dozen or two moves
if the other player is cooperating, but then it throws in a 
defection. If the other player continues to cooperate, then defections 
become more frequent. But as long as Tranquilizer is maintaining an 
average payoff of at least 2.25 points per move, it will never defect 
twice in succession and it will not defect more than 
one-quarter of the time.
```